src/components/TransactionBlock/AssetBridgeTransactionBlock.tsx

```diff
@@ -170,6 +175,19 @@ const AssetBridgeTransactionBlock = ({
     accountAddress,
   ]);
 
+  const getBestRouteItem = (routes: Route[]) => {
+    let bestRoute = routes[0];
+    let minAmount = routes[0].gasCostUSD ? +routes[0].fromAmountUSD - +routes[0].gasCostUSD : Number.MAX_SAFE_INTEGER;
+
+    routes.forEach(({ gasCostUSD, fromAmountUSD }, index) => {
+      if (gasCostUSD) {
+        if (+fromAmountUSD - +gasCostUSD > minAmount) bestRoute = routes[index];
+      }
+    });
```

Comment:
can be much much simpler
```suggestion
    routes.forEach((route) => {
      const { gasCostUSD, fromAmountUSD } = route;
      if (gasCostUSD) {
        if (+fromAmountUSD - +gasCostUSD > minAmount) bestRoute = route;
      }
    });
```